src/backend/termwiz.rs

```diff
@@ -67,6 +67,11 @@ impl TermwizBackend {
     ///
     /// The backend will automatically enable raw mode and enter the alternate screen.
     ///
+    /// Most applications will use either [`stdout`](std::io::stdout) or
+    /// [`stderr`](std::io::stderr) as writer. See the [FAQ] to determine which one to use.
+    ///
+    /// [FAQ]: https://ratatui.rs/faq/#should-i-use-stdout-or-stderr
+    ///
```

Comment:
Termwiz handles this automatically (IIRC it uses the current TTY instead of Stderr / Stdout).
```suggestion
```

Reply:
Right, I copy pasted too quickly